launch.go

```diff
@@ -429,6 +429,10 @@ func launch(api screwdriver.API, buildID int, rootDir, emitterPath, metaSpace, s
 		"SD_STORE_URL":           fmt.Sprintf("%s/%s/", storeURL, "v1"),
 		"SD_UI_URL":              fmt.Sprintf("%s/", uiURL),
 		"SD_TOKEN":               buildToken,
+		"SD_CACHE_STRATEGY":      cacheStrategy,
```

Comment:
so wanna make sure, so the default value for the strategy is set at the executor level?

Reply:
yes